versions.sh

```diff
@@ -148,8 +148,8 @@ for version in "${versions[@]}"; do
 			"bookworm",
 			"bullseye",
 			(
-				"3.18",
-				"3.17"
+				"3.19",
+				"3.18"
```

Comment:
Might be worth adding our `empty` hack here to make the diff cleaner next time (not worth cycling this time IMO :heart:).